libraries/wavemap/include/wavemap/data_structure/volumetric/hashed_chunked_wavelet_octree_block.h

```diff
@@ -57,10 +54,10 @@ class HashedChunkedWaveletOctreeBlock {
   bool getNeedsPruning() const { return needs_pruning_; }
   void setNeedsThresholding(bool value = true) { needs_thresholding_ = value; }
   bool getNeedsThresholding() const { return needs_thresholding_; }
-  void setLastUpdatedStamp(Time stamp = Clock::now()) {
+  void setLastUpdatedStamp(Timestamp stamp = Time::now()) {
     last_updated_stamp_ = stamp;
   }
-  Time getLastUpdatedStamp() const { return last_updated_stamp_; }
```

Comment:
Short answer: yes :)
Long answer: This way of handling time internally in the framework works well with ROS time (incl. bag playback) in the sense that the two time sources are never mixed, converted, or compared.
* ROS time is only used in the ROS interface (wavemap_server) to convert pointclouds+timestamps+odometry --> pointcloud+pose.
* The library internally uses temporal information only when checking if the map changed since some event occurred (e.g. since it was last published) and always does this using the same clock, accessed through the `Time = std::chrono::steady_clock;` alias for consistency.